manifests/install.pp

```diff
@@ -1,20 +1,18 @@
 # See README.md for usage information
 class sssd::install (
+  $ensure                = $sssd::ensure,
```

Comment:
Just realised my error here - think I merged more than I should have from the earlier PR. Also probably explains why the variable wasn't picked up above.

I'll clean that up tomorrow.